app/views/diabetes-content/diabetes-2-diagnosis.html

```diff
@@ -42,21 +42,19 @@
 Then make another [GP appointment](https://nhsalpha.herokuapp.com/book-an-appointment/start) and take your list of questions with you.
 
 ###Dealing with confusion and shock after the diagnosis
-There’s also a lot of [information on diabetes](/XXX) available where you can read up on the condition.
+There’s also a lot of [information on diabetes](https://nhsalpha.herokuapp.com/diabetes-content/information-and-support-type-2-diabetes) available on:
```

Comment:
Thought more as in 'info on support groups' etc on that page we link to

On 2 November 2015 at 11:50, Mat Johnson <notifications@github.com> wrote:

> In app/views/diabetes-content/diabetes-2-diagnosis.html
> <https://github.com/nhsalpha/nhs_prototype_kit/pull/143#discussion_r43620534>
> :
>
> > @@ -42,21 +42,19 @@
> >  Then make another [GP appointment](https://nhsalpha.herokuapp.com/book-an-appointment/start) and take your list of questions with you.
> >
> >  ###Dealing with confusion and shock after the diagnosis
> > -There’s also a lot of [information on diabetes](/XXX) available where you can read up on the condition.
> > +There’s also a lot of [information on diabetes](https://nhsalpha.herokuapp.com/diabetes-content/information-and-support-type-2-diabetes) available on:
>
> "available through" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nhsalpha/nhs_prototype_kit/pull/143/files#r43620534>.
>